navigator_engine/tests/unit_tests/test_conditionals.py

```diff
@@ -32,9 +32,9 @@ def test_check_manual_confirmation(mock_engine, action_id, expected):
         'navigator-workflow-state': {
             'data': {
                 'completedTasks': [
-                    "task1",
-                    "task2",
-                    "task3"
+                    {"id": "task1", "completedAt": "2021-12-10 15:22:12.640480"},
```

Comment:
API uses different date format: `"Mon, 13 Dec 2021 16:42:47 GMT"`

Reply:
I copied this straight from your own API tests

Reply:
I can update with the main NAV-211 PR